common/unified-chart/values.yaml

```diff
@@ -11,9 +11,29 @@ nameSuffix: v2
 
 argoRollouts:
   enabled: true
-  activeMetadata:
-    labels:
-      role: active
+  # activeMetadata:
+  #   labels:
+  #     role: active
+  progressiveDelivery:
+    enabled: true
+    stableMetadata:
+      labels:
+        role: stable
+    canaryMetadata:
+      labels:
+        role: canary
+    blackBox:
+      url: http://nexus.default.svc.cluster.local:8080/blackbox
+    steps:
+      - pause: {}
+      - setWeight: 100
+      - pause: { duration: 60s }
+      - analysis:
+          templates:
+            - templateName: black-box
```

Comment:
the template name should be different for every service. i suggest concating it with include name